MAUI/ImageEditor/text.md

```diff
@@ -0,0 +1,134 @@
+---
+layout: post
+title: Text Annotation in .NET MAUI ImageEditor control | Syncfusion
+description: Learn here all about the text annotation feature of Syncfusion .NET MAUI ImageEditor(SfImageEditor) control.
+platform: .NET MAUI
+control: SfImageEditor
+documentation: ug
+---
+
+# Text Annotation in .NET MAUI ImageEditor (SfImageEditor)
+
+The ImageEditor control in .NET MAUI allows you to add text with customizable settings.
```

Comment:
The ImageEditor control in .NET MAUI allows you to add text using the AddText method with customizable settings.

Reply:
Modified